diff --git a/node_modules/press-ready/lib/ghostScript.js b/node_modules/press-ready/lib/ghostScript.js-e
similarity index 88%
rename from node_modules/press-ready/lib/ghostScript.js
rename to node_modules/press-ready/lib/ghostScript.js-e
index 050e45c..b0a8339 100644
--- a/node_modules/press-ready/lib/ghostScript.js
+++ b/node_modules/press-ready/lib/ghostScript.js-e
@@ -21,6 +21,11 @@ function isGhostscriptAvailable() {
 }
 exports.isGhostscriptAvailable = isGhostscriptAvailable;
 async function ghostScript({ inputPath, outputPath, pdfxDefTemplatePath = upath_1.default.join(ASSETS_DIR, 'PDFX_def.ps.mustache'), sourceIccProfilePath = upath_1.default.join(ASSETS_DIR, 'JapanColor2001Coated.icc'), grayScale = false, enforceOutline = false, boundaryBoxes = false, title = 'Auto-generated PDF (press-ready)', }) {
+    const numTreads = 8;
+    const pdfxVersion = 4;
+    const dpi = 300;
+    const bandBufferSpace = 500000000;
+    const bufferSpace = 1000000000;
     const workingDir = os_1.tmpdir();
     const id = uuid_1.v4();
     // ICC profile
@@ -40,7 +45,11 @@ async function ghostScript({ inputPath, outputPath, pdfxDefTemplatePath = upath_
             (shelljs_1.default.which('gswin32c') && 'gswin32c'))) ||
         'gs';
     const gsOptions = [
-        '-dPDFX',
+        `-dNumRenderingThreads=${numTreads}`,
+        `-dBandBufferSpace=${bandBufferSpace}`,
+        `-dBufferSpace=${bufferSpace}`,
+        `-dPDFX=${pdfxVersion}`,
+        //'-dPDFX',
         '-dBATCH',
         '-dNOPAUSE',
         '-dNOOUTERSAVE',
@@ -49,10 +58,10 @@ async function ghostScript({ inputPath, outputPath, pdfxDefTemplatePath = upath_
         '-dShowAnnots=false',
         '-dPDFSETTINGS=/prepress',
         '-dPrinted',
-        '-r600',
-        '-dGrayImageResolution=600',
-        '-dMonoImageResolution=600',
-        '-dColorImageResolution=600',
+        `-r${dpi}`,
+        `-dGrayImageResolution=${dpi}`,
+        `-dMonoImageResolution=${dpi}`,
+        `-dColorImageResolution=${dpi}`,
         `-sOutputFile=${outputPath}`,
     ];
     if (boundaryBoxes) {
